### DISCUSSION TITLE
Demonstrate use of ethers/wagmi in docs

### DISCUSSION THREAD

**Discussion by chrisco512** [rainbow-me/rainbowkit#353] [Feedback]
Started poking around with Rainbowkit and really love the tool. But after getting through the connect step in the docs and tweaking the styles a bit, I realized I did not understand how to interface it offhand with ethers.js. I'm sure it's an easy thing and I just need to dig in more, but I think the docs could use some mention of where the kit interfaces with ethers and wagmi and give a bit of instruction on how a dev can hook into those tools if need be.

**Comment by nickbytes** [comment]
Thanks for the feedback @chrisco255 ! There's definitely more we can do to help understand how RainbowKit + wagmi + ethers all work together. We're building some examples and constantly improving our docs, will add a discussion piece here this week the team to figure out how we can do better.

In the meantime, check out:
- wagmi: https://wagmi.sh/
- ethers: https://docs.ethers.io/v5/

Here's a way you can think about these tools together:
- Ethers: a javascript library to interact w/ Ethereum 
- wagmi: React hooks around Ethers to make developing w/ Ethers feel more React-y + add common functionality for building dapps.
- RainbowKit: uses wagmi to build the best wallet connection experience for developers (and users)

Once you've got your wallet connection flow configured w/ RainbowKit, everything else you need to build your app and interact with Ethereum is in wagmi.
